src/platforms/android/configuration/integrations/jetpack-compose.mdx

```diff
@@ -0,0 +1,165 @@
+---
+title: Jetpack Compose
+caseStyle: camelCase
+supportLevel: production
+sdk: sentry.java.compose
+description: "Learn more about the Sentry Compose integration."
+categories:
+  - mobile
+  - desktop
+---
+
+The `sentry-compose` library provides [Jetpack Compose](https://developer.android.com/jetpack/androidx/releases/compose) Navigation support for Sentry using the [SentryNavigationIntegration](https://github.com/getsentry/sentry-java/blob/524ee49b212c3f2eead20960c3c6825ccdbf8007/sentry-compose/src/androidMain/kotlin/io/sentry/compose/SentryNavigationIntegration.kt). The source can be found [on GitHub](https://github.com/getsentry/sentry-java/blob/main/sentry-compose/src/androidMain/kotlin/io/sentry/compose/).
+
+On this page, we get you up and running with Sentry's Compose Navigation Integration, so that it will automatically add a breadcrumb and start an idle transaction for each navigation event.
+
+## Install
+
+Sentry captures data by adding a `withSentryObservableEffect` extension function to `NavHostController`. To add the Navigation integration, initialize the [Android SDK](/platforms/android/), then add the `sentry-compose` dependency using Gradle:
+
+```groovy
+implementation 'io.sentry:sentry-android:{{ packages.version('sentry.java.android', '6.2.0') }}'
+implementation 'io.sentry:sentry-compose:{{ packages.version('sentry.java.compose', '6.2.0') }}'
+```
+
+## Configure
+
+Configuration should happen in the respective `@Composable` function, once you obtain an instance of `NavHostController`:
+
+```kotlin
+import androidx.navigation.compose.NavHost
+import androidx.navigation.compose.rememberNavController
+import io.sentry.compose.withSentryObservableEffect
+
+val navController = rememberNavController().withSentryObservableEffect(
+  enableNavigationBreadcrumbs = true, // enabled by default
+  enableNavigationTracing = true  // enabled by default
+)
+NavHost(
+  navController = navController
+) {
+  ...
+}
+```
+
+By default, the navigation transaction finishes automatically after it reaches the specified [idleTimeout](/platforms/android/configuration/options/#idle-timeout) and all of its child spans are finished. The `idleTimeoout` defaults to `3000` milliseconds (three seconds), but you can customize it to your needs.
```

Comment:
Maybe we don't mention the `default` value here since it's documented already under the `idleTimeout` link, a single source of truth.